build/build.sh

```diff
@@ -33,53 +33,19 @@ pandoc --verbose \
   --bibliography=$BIBLIOGRAPHY_PATH \
   --csl=$CSL_PATH \
   --metadata link-citations=true \
-  --mathjax \
   --include-after-body=build/themes/default.html \
-  --include-after-body=build/plugins/analytics.html \
+  --include-after-body=build/plugins/table-scroll.html \
   --include-after-body=build/plugins/anchors.html \
-  --include-after-body=build/plugins/hypothesis.html \
+  --include-after-body=build/plugins/accordion.html \
+  --include-after-body=build/plugins/tooltips.html \
+  --include-after-body=build/plugins/jump-to-first.html \
+  --include-after-body=build/plugins/link-highlight.html \
+  --include-after-body=build/plugins/table-of-contents.html \
+  --include-after-body=build/plugins/lightbox.html \
+  --mathjax \
+  --variable math="" \
+  --include-after-body=build/plugins/math.html \
   --output=output/manuscript.html \
   $INPUT_PATH
 
-# Create PDF output (unless BUILD_PDF environment variable equals "false")
-if [ "$BUILD_PDF" != "false" ]; then
-  echo "Exporting PDF manuscript"
-  if [ -L images ]; then rm images; fi  # if images is a symlink, remove it
-  ln -s content/images
-  pandoc \
-    --from=markdown \
-    --to=html5 \
-    --pdf-engine=weasyprint \
-    --pdf-engine-opt=--presentational-hints \
-    --filter=pandoc-fignos \
-    --filter=pandoc-eqnos \
-    --filter=pandoc-tablenos \
-    --bibliography=$BIBLIOGRAPHY_PATH \
-    --csl=$CSL_PATH \
-    --metadata link-citations=true \
-    --webtex=https://latex.codecogs.com/svg.latex? \
-    --include-after-body=build/themes/default.html \
-    --output=output/manuscript.pdf \
-    $INPUT_PATH
-  rm images
-fi
-
-# Create DOCX output (if BUILD_DOCX environment variable equals "true")
-if [ "$BUILD_DOCX" = "true" ]; then
-  echo "Exporting Word Docx manuscript"
-  pandoc --verbose \
-    --from=markdown \
-    --to=docx \
-    --filter=pandoc-fignos \
-    --filter=pandoc-eqnos \
-    --filter=pandoc-tablenos \
-    --bibliography=$BIBLIOGRAPHY_PATH \
-    --csl=$CSL_PATH \
-    --metadata link-citations=true \
-    --reference-doc=build/themes/default.docx \
-    --resource-path=.:content \
-    --output=output/manuscript.docx \
-    $INPUT_PATH
-fi
-
```

Comment:
Must not delete PDF and DOCX build commands.

Reply:
These were both mistakes. Had removed them to speed up the build during testing, forgot to add them back in. Fixed.